### ISSUE TITLE
[iOS] No spacing between lines if content exceeds the space for one line

### ISSUE THREAD

**Issue by ghost** [subsurface/subsurface#3263]
### Describe the issue:

- [ x] Bug
- [ ] Change request
- [ ] New feature request
- [ ] Discussion request

### Issue long description:

If the content of Suit or Tags exceeds the available space for one line, there is no spacing between lines in portrait mode:
![Subsurface_iOS_multiple_lines_glitch](https://user-images.githubusercontent.com/3070848/120915108-328c7700-c6a2-11eb-8bb1-89e23b9ca650.jpeg)

### Operating system:

iOS 14.5

iPhone 11

### Subsurface version:

3.2.1(5.0.1)

unknown

official release

### Steps to reproduce:

- in a dive add a dive suit description too long for one line in portrait mode
- in a dive add tags too long for one line in portrait mode

### Current behavior:

No spacing between lines.

### Expected behavior:

Proper spacing between lines.

**Comment by dirkhh** [comment]
Yeah, visually not ideal. Sometimes switching to a slightly smaller font helps - but given that we support quite literally something like 7000 different devices, on some of them the spacing seems to look better, on some of them it looks worse.
